src/components/posts/postForm.js

```diff
@@ -32,22 +31,19 @@ export const PostForm = (props) => {
     const populateFormValues = post => {
       titleRef.current.value = post.title
       categoryRef.current.value = post.category_id
-      imageRef.current.value = post.image
+      imageRef.current.value = post.image_url
       contentRef.current.value = post.content
 
       // HTML date inputs take their value in the format YYYY-MM-DD
```

Comment:
This comment no longer refers to anything... can we get rid of it?

Reply:
This is removed